hudi-common/src/main/java/org/apache/hudi/common/engine/HoodieReaderContext.java

```diff
@@ -67,6 +70,7 @@ public abstract class HoodieReaderContext<T> {
    * file.
    *
    * @param filePath       {@link Path} instance of a file.
+   * @param isLogFile      Whether this is a log file.
    * @param start          Starting byte to start reading.
```

Comment:
Can we infer the `isLogFile` flag directly from the file extension?

Reply:
Fixed.